.github/workflows/agw-coverage.yml

```diff
@@ -0,0 +1,242 @@
+        uses: ./.github/workflows/composite/maximize-build-space
+      - name: Setup Bazel Base Image
+        uses: addnab/docker-run-action@4f65fabd2431ebc8d299f8e5a018d79a769ae185 # pin@v3
+        with:
+          image: ${{ env.BAZEL_BASE_IMAGE }}
+          options: --pull always
+          # Run a simple echo command to pull down the image. This makes it a bit more clear how much time is spent on building Magma and not pulling down the image.
+          run: |
+            echo "Pulled the bazel base image!"
+            bazel # pull down bazel, if bazel download fails we can fail before we do all the lengthy work below
+      - name: Run C/C++ coverage with Bazel
+        if: always()
+        id: bazel-cc-codecoverage
+        uses: addnab/docker-run-action@4f65fabd2431ebc8d299f8e5a018d79a769ae185 # pin@v3
+        with:
+          image: ${{ env.BAZEL_BASE_IMAGE }}
+          # TODO: Remove work-around mount of Github workspace to /magma (https://github.com/addnab/docker-run-action/issues/11)
+          options: -v ${{ github.workspace }}:/workspaces/magma/ -v ${{ github.workspace }}/lte/gateway/configs:/etc/magma
+          run: |
+            cd $MAGMA_ROOT
+            bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION_C_CPP }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}"
+            # Collecting coverage with Bazel can be slow. We can follow this thread to see if this can be improved: https://github.com/bazelbuild/bazel/issues/8178
+            # Coverage in bazel is flaky for remote caches - the flags below are helping. See GH13026 for details.
+            bazel coverage \
+              --profile=Bazel_test_cc_coverage_profile \
+              --experimental_split_coverage_postprocessing --experimental_fetch_all_coverage_outputs --remote_download_outputs=all \
+              //orc8r/gateway/c/...:* //lte/gateway/c/...:*
+            # copy out coverage information into magma so that it's accessible from the CI node
+            cp bazel-out/_coverage/_coverage_report.dat $MAGMA_ROOT
+      - name: Upload code coverage
+        if: always()
+        id: c-cpp-codecov-upload
+        uses: codecov/codecov-action@d9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70 # pin@v3.1.1
+        with:
+          flags: c_cpp
+      - name: Publish bazel profile
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # pin@v3
+        if: |
+          always() &&
+          github.repository_owner == 'magma' &&
+          github.ref_name == 'master'
+        with:
+          name: Bazel test C/C++ coverage profile
+          path: Bazel_test_cc_coverage_profile
+      - name: Extract commit title
+        # yamllint enable
+        if: failure() && github.event_name == 'push'
+        id: commit
+        run: |
+          str="$(jq '.head_commit.message' $GITHUB_EVENT_PATH)"    # get the head_commit message
+          echo ::set-output name=title::${str%%\\n*} | tr -d '"'
```

Comment:
This seems to be unnecessary. The information is probably available in the event anyway.
Or why would we need that?

Reply:
Yes the information is available when you click on the run and then navigate to the PR, but it is more convenient and this is the standard formatting we have for all Slack notifications with this action. I would keep it for consistency and convenience.

Reply:
Remove now anyway since it will not be sending to that channel.